.travis.yml

```diff
@@ -57,3 +64,5 @@ deploy:
     on:
         repo: chakki-works/doccano
         branch: master
+
+        # Use container-based infrastructure for quicker build start-up
```

Comment:
Nit: remove unnecessary comment.

Reply:
This comment was previously here. Should I still delete?